src/Factory.php

```diff
@@ -204,47 +203,47 @@ public static function toFile($filename, $config)
     /**
      * Set reader plugin manager
      *
-     * @param ReaderPluginManager $readers
+     * @param ContainerInterface $readers
      * @return void
      */
-    public static function setReaderPluginManager(ReaderPluginManager $readers)
+    public static function setReaderPluginManager(ContainerInterface $readers)
     {
         static::$readers = $readers;
     }
 
     /**
      * Get the reader plugin manager
      *
-     * @return ReaderPluginManager
+     * @return ContainerInterface
```

Comment:
Maybe we should have here also `StandaloneReaderPluginManager`, so generic one and specific/default?

Yes, I know that `StandaloneReaderPluginManager implements ContainerInterface`.